artiq/firmware/libboard_artiq/pca9548.rs

```diff
@@ -0,0 +1,13 @@
+use i2c;
+
+pub fn select(busno: u8, address: u8, channels: u8) -> Result<(), &'static str> {
```

Comment:
Move to i2c.rs? It's a bit weird to have such a small file and also many boards with i2c also have a pca9548.

Reply:
For traits? Why no embedded_hal? There is also a `xca9548a` crate.

Can this be moved into a follow-up PR?

Reply:
* embedded_hal (and other things) didn't exist when the ARTIQ firmware was initially written.
* I don't know if those libraries are good - needs review and testing.
* Someone needs to do the work of refactoring the firmware to use those libs, and there are often more important things to do.